lib/inspec/resources/podman_image.rb

```diff
@@ -0,0 +1,144 @@
+require "inspec/resources/command"
+require_relative "docker_object"
+
+module Inspec::Resources
+  class PodmanImage < Inspec.resource(1)
+    include Inspec::Resources::DockerObject
+    name "podman_image"
+    supports platform: "unix"
+
+    desc "InSpec core resource to retrieve information about podman image"
+
+    example <<~EXAMPLE
+      describe podman_image("docker.io/library/busybox") do
+        it { should exist }
+        its("repo_tags") { should include "docker.io/library/busybox:latest" }
+        its("size") { should eq 1636053 }
+        its("resource_id") { should eq "docker.io/library/busybox:latest" }
+      end
+
+      describe podman_image("docker.io/library/busybox:latest") do
+        it { should exist }
+      end
+
+      describe podman_image(repo: "docker.io/library/busybox", tag: "latest") do
+        it { should exist }
+      end
+
+      describe podman_image(id: "3c19bafed223") do
+        it { should exist }
+      end
+    EXAMPLE
+
+    attr_reader :opts, :inspect_info
+
+    def initialize(opts)
+      skip_resource "The `podman_image` resource is not yet available on your OS." unless inspec.os.unix?
+      opts = { image: opts } if opts.is_a?(String)
+      @opts = sanitize_options(opts)
+      @inspect_info = get_inspect_info
```

Comment:
In `get_inspec_info` method, we actually getting the image data using the `podman image inspect` command, so naming that method to something specific might be helpful. and maybe also the `@inspect_info` to `@image_info`

Reply:
Any suggestion for the method and attributes name?

Reply:
image_inspect_info?